commands/commands.go

```diff
@@ -182,9 +186,10 @@ func handleCommonErrors(err error) {
 			headline = "The API didn't send a response."
 			subtext = "Please check your connection using 'gsctl ping'. If your connection is fine,\n"
 			subtext += "please try again in a few moments."
-		case config.IsUnableToRefreshTokenErrorr(err):
-			headline = "Unable to refresh your SSO Token."
-			subtext = "Please try loging in again using: gsctl login --sso"
+		case oidc.IsRefreshError(err):
+			headline = "Unable to refresh your SSO token."
+			subtext = err.Error() + "\n"
+			subtext += "Please try loging in again using: gsctl login --sso"
```

Comment:
This could be placed immediately before or after with `case oidc.IsAuthorizationError(err):`. Minor.